src/main/java/seedu/duke/util/StringConstants.java

```diff
@@ -109,6 +116,12 @@
             + "Available commands: exit, add, del, list, mark, save, help";
     public static final String HELP_EXCEPTION = "Sorry, but no help exists for that command.";
 
+    public static final String OPTION_HELP = "Set customized configuration\n"
+            + "Format to set an option: option CONFIGURATION_GROUP=NEW_VALUE\n"
```

Comment:
I'm guessing there might be future configuration groups that use NEW_VALUE other than boolean type? If not, maybe BOOLEAN might be clearer.

Reply:
Currently the new value is not boolean type, but String instead. But we need to define the legal value set to avoid unpredictable behaviors